ScanAnalysis/live_watch/scan_analysis_gui/app/ScAnalyzer.py

```diff
@@ -211,17 +230,18 @@ def handle_error(self, error_message: str) -> None:
         '''
         self.write_to_log_display(f"Error: {error_message}")
 
-    def set_default_inputs(self) -> None:
+    def event_overwrite_button_clicked(self) -> None:
         '''
-        Assign default values to text fields.
-        :return: No return.
-        :rtype: None
+        Actions performed when Overwrite button is clicked.
+
+        Returns
+        -------
+        None
         '''
-        # set default scan date
-        today = date.today()
-        self.ui.inputYear.setText(str(today.year))
-        self.ui.inputMonth.setText(str(today.month))
-        self.ui.inputDay.setText(str(today.day))
+        if self.ui.buttonOverwrite.isChecked():
+            self.ui.inputStartScan.setEnabled(True)
+        else:
+            self.ui.inputStartScan.setEnabled(False)
```

Comment:
If isChecked() returns a bool, then this can simply be:
```suggestion
        self.ui.inputStartScan.setEnabled(self.ui.buttonOverwrite.isChecked())
```

Reply:
Good catch. Fixed. Thanks.